spyder/utils/tests/test_conda.py

```diff
@@ -65,8 +65,17 @@ def test_get_conda_root_prefix():
 
 @pytest.mark.skipif(not running_in_ci(), reason="Only meant for CIs")
 def test_find_conda():
+    # Temporarily remove CONDA_EXE and MAMBA_EXE, if present
+    conda_exe = os.environ.pop('CONDA_EXE', None)
+    mamba_exe = os.environ.pop('MAMBA_EXE', None)
```

Comment:
I'd prefer to have a parametrized test for this so that we can check if `find_conda` works with and without these env vars present.